src/DiIiS-NA/D3-GameServer/GSSystem/ItemsSystem/Item.cs

```diff
@@ -1088,7 +1088,7 @@ public virtual void OnRequestUse(Player player, Item target, int actionId, World
                                 break;
                         }
 
-                        it = items[RandomHelper.Next(@base, @base + 1)]; // FIXME: this random always returns @base
+                        it = items[RandomHelper.Next(@base, @base + 2)];
```

Comment:
according to the code above (base is even - 2, 4, 6, etc.) I think it should be + 2
below the same